superset/utils/log.py

```diff
@@ -36,31 +36,32 @@ def log(
         pass
 
     def log_this(self, f: Callable[..., Any]) -> Callable[..., Any]:
+        from superset.views.core import get_form_data
```

Comment:
it would be great to add a unit test here to prevent the regressions

Reply:
@bkyryliuk I've added a unit test per your suggestion. Note rather than mocking the `log_this` wrapped function and callback opted to call the `explore_json` endpoint and verified the logged database record.